app/controllers/users_controller.rb

```diff
@@ -127,6 +127,7 @@ def bulk_destroy
   # shows printable login instructions for the user
   def login_instructions
     @site_url = admin_mode? ? basic_root_url : mission_root_url
+    @config_qr = ODKConfigGenerator.new().generate_odk_config(@user.login, flash[:password], @site_url)
```

Comment:
Interesting. I'd be keen to discuss this offline. But for now yeah maybe best to keep consistent?